reference/shipping_provider.yml

```diff
@@ -1984,14 +1984,12 @@ components:
                     type: string
             customer_id:
               type: integer
-        cart_id:
-          type: string
         store_id:
           type: string
         request_context:
           type: object
           title: Request Context
-          description: A collection of Reference Value objects.
+          description: A collection of Reference Value objects. Two examples of this is the cart_id and the channel_id.
```

Comment:
🍻 Would we want to mark these terms in any special way @bc-andreadao ? Unsure if we have a common approach here.

For example `cart_id` and `channel_id` would be how I would do it in this GitHub comment 😆

Reply:
@theromulans I changed it so that the examples show up in the schema: (looks better in my opinion) 

![examples](https://user-images.githubusercontent.com/96258747/204383635-025fc175-f755-4cfb-b9a2-72390caefc6c.png)

Reply:
That's awesome, thanks @bc-andreadao.